### ISSUE TITLE
Add a CNN for image recognition

### ISSUE THREAD

**Issue by the-ethan-hunt** [NJACKWinterOfCode/nervy-notebooks#5]
Write an `IPython notebook` for a convolutional neural network implementation for image recognition. It is highly requested not to use MNIST or CIFAR10 dataset

**Comment by vanshsoni** [comment]
Hello,
Can I use tensorflow for adding CNN for image recognition

**Comment by the-ethan-hunt** [comment]
Yes, you are free to use any framework/library. But while doing that, please name your `IPython notebook` as `CNN_using_tensorflow`.

**Comment by vanshsoni** [comment]
@the-ethan-hunt I am working on this
I will update you tomorrow 👍
Merry Christmas

**Comment by the-ethan-hunt** [comment]
@vanshsoni , I would suggest you to use `Torch` here for better results

**Comment by vanshsoni** [comment]
ok I will try to do that but I almost created using tensorflow
